third_party/qpg_sdk/qpg_sdk.gni

```diff
@@ -16,31 +16,44 @@ import("//build_overrides/chip.gni")
 import("//build_overrides/jlink.gni")
 import("//build_overrides/mbedtls.gni")
 import("//build_overrides/openthread.gni")
-import("//build_overrides/qpg6100_sdk.gni")
+import("//build_overrides/qpg_sdk.gni")
 
 declare_args() {
-  # Location of the QPG6100 SDK.
-  qpg6100_sdk_root = "${chip_root}/third_party/qpg_sdk/repo"
+  # Location of the QPG SDK.
+  qpg_sdk_root = "${chip_root}/third_party/qpg_sdk/repo"
+
+  # Target IC for QPG SDK
+  qpg_target_ic = "qpg6100"
 }
 
-qpg6100_sdk_root_env = getenv("QPG6100_SDK_ROOT")
-if (qpg6100_sdk_root_env != "") {
-  qpg6100_sdk_root = qpg6100_sdk_root_env
+qpg_sdk_root_env = getenv("QPG_SDK_ROOT")
+if (qpg_sdk_root_env != "") {
+  qpg_sdk_root = qpg_sdk_root_env
 }
+assert(qpg_sdk_root != "", "qpg_sdk_root must be specified")
 
-assert(qpg6100_sdk_root != "", "qpg6100_sdk_root must be specified")
+qpg_target_ic_env = getenv("QPG_TARGET_IC")
+if (qpg_target_ic_env != "") {
+  qpg_target_ic = qpg_target_ic_env
+}
+assert(qpg_target_ic != "", "qpg_target_ic must be specified")
```

Comment:
Please make the explicit arguments override the environment, not the opposite. It's going to be terribly confusing if builddir/args.gn contains a value that is thrown away in favor of an environment variable.

I would actually recommend to remove the environment value handling entirely, and force configuration into the command line or build arguments, but I do understand the convenience it provides (the problems are two: (1) ninja won't automatically regenerate on environment changes so use of the environment can cause incorrect incremental build outputs and (2) environment variables, being cross-process global variables, have all the problems of global variables such as implicit coupling).

Reply:
@mspang Thanks for the feedback. The main intent here was to swap out the internal SDK easily during development. The arguments cover that purpose equally.
Removed the env overrides in favor of using the arguments.